src/cache/cache_las.c

```diff
@@ -35,6 +35,17 @@ __las_restore_isolation(WT_SESSION_IMPL *session, WT_TXN_ISOLATION saved_isolati
     session->txn.isolation = saved_isolation;
 }
 
+/*
+ * __las_store_time_pair --
+ *     Store the time pair to use for the lookaside inserts.
+ */
+static void
+__las_store_time_pair(WT_SESSION_IMPL *session, wt_timestamp_t timestamp, uint64_t txnid)
+{
+    S2C(session)->cache->orig_timestamp_to_las = timestamp;
```

Comment:
Why store the time pair info at the global level? Why not at session-level. What is the need of having time pair info at the global level and make it a critical section?

Reply:
Moved to `WT_SESSION_IMPL` structure.